cmd/access/node_builder/access_node_builder.go

```diff
@@ -629,8 +643,24 @@ func (builder *FlowAccessNodeBuilder) BuildExecutionSyncComponents() *FlowAccess
 				}
 
 				trackerDir := filepath.Join(builder.executionDataDir, "tracker")
+				var storageDB storagedb.StorageDB
+				if builder.pebbleDBExecutionDataEnabled {
+					storageDB, err = storagedb.NewPebbleDBWrapper(trackerDir, nil)
+					if err != nil {
+						return nil, fmt.Errorf("could not create tracker NewPebbleDBWrapper: %w", err)
+					}
+
+				} else {
+					options := badgerds.DefaultOptions
+					options.Options = badger.LSMOnlyOptions(trackerDir)
+					storageDB, err = storagedb.NewBadgerDBWrapper(trackerDir, &options)
```

Comment:
since you have to pass `trackerDir ` anyway, you could move the options override into `trackerDir `
```suggestion
					options := badgerds.DefaultOptions
					storageDB, err = storagedb.NewBadgerDBWrapper(trackerDir, &options)
```